app/views/waste_calendar/new.html.erb

```diff
@@ -44,7 +43,7 @@
           </h2>
           <p>
             Bitte kopieren Sie die CSV-Daten der Touren aller Abfallarten hinein.<br /><br />
-            <%= image_tag("example_pick_up_times.png") %><br /><br />
+            <%= image_tag("example_pick_up_times.png", :class => "img-fluid") %><br /><br />
```

Comment:
please use `class: "img-fluid"` as this is current best practice syntax and also better to read for you i think

Reply:
Ist angepasst